app/scripts/models/auth_brokers/fx-sync.js

```diff
@@ -19,21 +13,38 @@ define([
   'underscore',
   'models/auth_brokers/base',
   'models/auth_brokers/mixins/channel',
-  'lib/auth-errors',
-  'lib/channels/fx-desktop-v1',
-  'lib/url'
-], function (Cocktail, _, BaseAuthenticationBroker, ChannelMixin, AuthErrors,
-  FxDesktopChannel, Url) {
+  'lib/auth-errors'
+], function (Cocktail, _, BaseAuthenticationBroker, ChannelMixin, AuthErrors) {
   'use strict';
 
-  var FxDesktopAuthenticationBroker = BaseAuthenticationBroker.extend({
-    type: 'fx-desktop-v1',
-    _commands: {
-      CAN_LINK_ACCOUNT: 'can_link_account',
-      CHANGE_PASSWORD: 'change_password',
-      DELETE_ACCOUNT: 'delete_account',
-      LOADED: 'loaded',
-      LOGIN: 'login'
+  var proto = BaseAuthenticationBroker.prototype;
+
+  var FxSyncAuthenticationBroker = BaseAuthenticationBroker.extend({
+    type: 'fx-sync',
+
+    /**
+     * _commands must be overridden with an object that contains:
+     * {
+     *   CAN_LINK_ACCOUNT: <specify in subclass>,
+     *   CHANGE_PASSWORD: <specify in subclass>,
+     *   DELETE_ACCOUNT: <specify in subclass>,
+     *   LOADED: <specify in subclass>,
+     *   LOGIN: <specify in subclass>\
+     * }
+     */
+    _commands:null,
```

Comment:
I'm thinking we need to set the `key-spacing` rule in our .eslintrc file (or ideally upstream in the **eslint-config-fxa** module).

```js
"key-spacing": [2, { "beforeColon": false, "afterColon": true }]
```

If I try changing that locally against *master* branch, I get the following output:
```sh
➜  fxa-content-server git:(master) ✗ grunt eslint --force
Running "eslint:grunt" (eslint) task


Running "eslint:app" (eslint) task

/Users/pdehaan/dev/fxa-content-server_pd/fxa-content-server/app/scripts/views/force_auth.js
  71:17  error  Extra space before value for key "email"  key-spacing

/Users/pdehaan/dev/fxa-content-server_pd/fxa-content-server/app/tests/spec/models/auth_brokers/oauth.js
  107:26  error  Extra space before value for key "redirect"  key-spacing
  148:26  error  Extra space before value for key "redirect"  key-spacing
  169:26  error  Extra space before value for key "redirect"  key-spacing

✖ 4 problems (4 errors, 0 warnings)

Warning: Task "eslint:app" failed. Used --force, continuing.

Running "eslint:tests" (eslint) task


Running "eslint:scripts" (eslint) task


Running "eslint:server" (eslint) task


Done, but with warnings.
```